flask_ckeditor/__init__.py

```diff
@@ -163,7 +163,82 @@ def load_code_theme():
             <script>hljs.initHighlightingOnLoad();</script>''' % (css_url, js_url))
 
 
-class CKEditor(object):
+class _CKEditor5(object):
+    """The class implement functions for Jinja2 template."""
+
+    def __init__(self):
+        self.editor_dict = {'classic':'ClassicEditor',
+                        'inline':'InlineEditor', 
+                        'balloon':'BalloonEditor',
+                        'balloon-block':'BalloonEditor',
+                        'decoupled-document':'DecoupledEditor'}
+
+    def check_editor_type(self, editor_type):
+        editor_type = editor_type or current_app.config['CKEDITOR_EDITOR_TYPE']
+        if editor_type not in self.editor_dict.keys():
+            warnings.warn('The provided editor_type string was invalid, '
+                          'it should be one of classic/inline/balloon/balloon-block/decoupled-document.')
+            editor_type = 'classic'
+        return editor_type
+
+    def extra_plugin(self, file_uploader=None):
+        plugin = []
+
+        if file_uploader is not None:
+            plugin.append('''ckfinder:{uploadUrl: "%s"}''' % (file_uploader))
+
+        return ',\n\t\t\t'.join(plugin)
+       
+
+    def load(self, custom_url=None, editor_type=None, serve_local=None, version='28.0.0'):
+        """Load CKEditor resource from CDN or local."""
+
+        self.editor_type = self.check_editor_type(editor_type)
+        url = 'https://cdn.ckeditor.com/ckeditor5/%s/%s/ckeditor.js' % (version, self.editor_type)
+
+        if custom_url:
+            url = custom_url
+
+        return Markup('<script src="%s"></script>' % url)
+
+    def config(self, name='ckeditor', custom_config='', **kwargs):
+        """Config CKEditor."""
+
+
+        file_uploader = kwargs.get('file_uploader', current_app.config['CKEDITOR_FILE_UPLOADER'])
+
+        if file_uploader != '':
```

Comment:
Why not something like `if not file_uploader` instead of `if file_uploader != ''` ?

Reply:
By the way, nice work, @WebberMao